package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cyclonedx/cdxgen",
-  "version": "10.4.1",
+  "version": "10.5.0",
```

Comment:
Please do not change the version, since it requires more manual changes at multiple locations.